src/screens/DiscoveryScreenV2/index.js

```diff
@@ -225,11 +326,11 @@ const styles = StyleSheet.create({
     paddingHorizontal: 24,
     paddingVertical: 4,
     fontFamily: fonts.inter[600]
-  }
+  },
+  headerTitle: {fontSize: 16, fontFamily: fonts.inter[600], textAlign: 'center'}
 });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `headerTitle` style is added. Ensure that this style is applied correctly where needed.



<!-- This is an auto-generated comment by OSS CodeRabbit -->